.github/workflows/build-pr.yml

```diff
@@ -14,6 +14,17 @@ jobs:
       - uses: actions/checkout@v2
       - name: build image
         run: docker build --target=current -t documentation:latest .
+      - name: copy files to host
+        run: docker build --target=deploy-source --output=./_site .
+      - name: check for broken links
+        uses: chabad360/htmlproofer@master
+        with:
+          directory: ./_site
+          # for available options, refer to:
+          # - https://github.com/gjtorikian/html-proofer
+          # - https://github.com/gjtorikian/html-proofer/blob/main/bin/htmlproofer
+          arguments: --disable-external --internal-domains="docs.docker.com,docs-stage.docker.com,localhost:4000" --file-ignore="/^./_site/engine/api/.*$/,./_site/registry/configuration/index.html,./_site/engine/reference/commandline/buildx_build/index.html,./_site/engine/reference/commandline/buildx_create/index.html" --url-ignore="/^/docker-hub/api/latest/.*$/,/^/engine/api/v.+/#.*$/,/^/glossary/.*$/"
```

Comment:
We need to remove the buildx excludes once they're fixed (https://github.com/docker/buildx/pull/765 is merged, but only on master)